packages/components/src/Chip/Chip.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import styles from "./Chip.css";
+import { Avatar, AvatarProps } from "../Avatar";
+import { Text } from "../Text";
+import { ButtonDismiss } from "../ButtonDismiss";
+
+interface DismissibleAction {
+  ariaLabel: string;
+  onDismiss(): void;
+}
+
+interface ChipProps {
+  label: string;
+  avatar?: Exclude<AvatarProps, "size">;
+  dismissAction: DismissibleAction;
+}
+
+export function Chip({
+  label,
+  avatar,
+  dismissAction: { ariaLabel, onDismiss },
+}: ChipProps) {
+  return (
+    <span className={styles.chip}>
+      {avatar && <Avatar {...avatar} size="small" />}
+      <Text>{label}</Text>
+      <ButtonDismiss ariaLabel={ariaLabel} onClick={onDismiss} size="small" />
```

Comment:
Extra convenience thing. Since the label is a `String`, we can pretty much guess what the aria label would be. I.e., `"Remove" + label` = `"Remove Darryl Tec"`. Then at that point, the aria label can be optional or, dare I say, not exists?.

Reply:
While I agree it should always likely be `Remove + label`, I believe allowing whoever to specify the string works best, especially considering localization and what not.

Reply:
Good point!! Forgot about the localization thing